spec.md

```diff
@@ -134,41 +134,59 @@ the paths to label objects which can be found underneath the group:
 
 Unlisted groups MAY be labels.
 
-### "label""
+### "image-label" metadata
 
+Groups containing the `image-label` dictionary represent an image segmentation
+in which each unique pixel value represents a separate segmented object.
+`image-label` groups MUST also contain `multiscales` metadata and the two
+"datasets" series MUST have the same number of entries.
```

Comment:
I'm a bit confused where in the hierarchy the `image-label` metadata goes compared with the `multiscales` metadata.

Re:

```
└── labels
        │
        ├── .zgroup           # The labels group is a container which holds a list
        ├── .zattrs           # of labels to make the objects easily discoverable,
        │                     # All labels will be listed in `.zattrs` e.g. `{ "labels": [ "original/0" ] }`
        │                     # Each dimension of the label `(t, c, z, y, x)` should be either the same as the
        │                     # corresponding dimension of the image, or `1` if that dimension of the label
        │                     # is irrelevant.
        │
        └── original          # Intermediate folders are permitted but not necessary
            │                 # and currently contain no extra metadata.
            │
            └── 0             # Multiscale, labeled image. The name is unimportant but is registered in "labels".
                ├── .zgroup   # Each labeled image is also a multiscaled image, and therefore a Zarr group.
                ├── .zattrs   # Metadata of the related image and as well as display information under the "label" key.
                │
                ├── 0         # Each multiscale level is stored as a separate Zarr array, as above.
                │   ...
                └── n
```

The "group that contains image-label dictionary" is the lower-level .zgroup (sibling of the .zattrs that contains the "image-label" metadata)? So this group "MUST also contain `multiscales` metadata"?